ores/ores.py

```diff
@@ -30,7 +28,7 @@ def main():
         sys.stderr.write(USAGE)
         sys.exit(1)
     elif sys.argv[1] in ("-h", "--help"):
-        sys.stderr.write(__doc__ + "\n")
+        sys.stderr.write(__doc__ % { "usage": USAGE })
```

Comment:
Hmm.  It seems like we shouldn't make this distinction.  `./utility` is intended to be run under the dev environment and operate as closely as to the `ores` utility as possible.  Maybe you are hoping to avoid user-error somehow?